### ISSUE TITLE
Scope-level filters should take precedence over 'global' filters

### ISSUE THREAD

**Issue by MaKleSoft** [Polymer/polymer-expressions#49]
Currently, filters defined on PolymerExpressions.prototype take precedence over filters defined on the current model/scope. I think it should be the other way around as the current scope obviously has a higher specificity than the 'global' filter scope.

**Comment by pflannery** [comment]
+1

I think its a simple switch of logic [at this location](https://github.com/Polymer/polymer-expressions/blob/master/src/polymer-expressions.js#L195-L205) , I'm happy to do a PR on this

```js
var context = model,
    fn = context[this.name];

if (!fn) {
    fn = filterRegistry[this.name];
    if (!fn) {
        console.error('Cannot find function or filter: ' + this.name);
        return;
    }
}
```

The code above also lets filters in the ```PolymerExpressions.prototype``` receive the model as the context because currently it's being set to ```undefined``` which defaults the context to the window object.

**Comment by jmesserly** [comment]
thanks @pflannery ... go for it!

**Comment by jmesserly** [comment]
fixed in #60, :+1:
